src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -0,0 +1,191 @@
+            }
+            break;
+          default:
+            throw new Error("INVALID NATURE OF CONTROL TYPE");
+      }
+    }
+  });
+};
+
+enum BoTypes {
+  BO_NATURE_OF_CONTROL = "BoNatureOfControl",
+  TRUST_NATURE_OF_CONTROL = "TrustNatureOfControl",
+  NON_LEGAL_NATURE_OF_CONTROL = "NonLegalNatureOfControl",
+}
+
+export enum controlMap {
+  OWNERSHIP_MORE_THAN_25_PERCENT_SHARE_ENTITY = 'ownership-of-shares-more-than-25-percent-registered-overseas-entity',
+  VOTING_RIGHT_MORE_THAN_25_PERCENT_SHARE_ENTITY = 'voting-rights-more-than-25-percent-registered-overseas-entity',
+  RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS = 'right-to-appoint-and-remove-directors-registered-overseas-entity',
+  SIGNIFICANT_INFLUENCE_OR_CONTROL_ENTITY = 'significant-influence-or-control-registered-overseas-entity',
+  RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_TRUST = 'right-to-appoint-and-remove-directors-as-trust-registered-overseas-entity',
+  SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_TRUST = 'significant-influence-or-control-as-trust-registered-overseas-entity',
+  VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_TRUST ='voting-rights-more-than-25-percent-as-trust-registered-overseas-entity',
+  OWNERSHIP_MORE_THAN_25_PERCENT_AS_TRUST = 'ownership-of-shares-more-than-25-percent-as-trust-registered-overseas-entity',
+  SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM = 'significant-influence-or-control-as-firm-registered-overseas-entity',
+  RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_FIRM = 'right-to-appoint-and-remove-directors-as-firm-registered-overseas-entity',
+  VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_FIRM = 'voting-rights-more-than-25-percent-as-firm-registered-overseas-entity',
+  OWNERSHIP_MORE_THAN_25_PERCENT_AS_FIRM = 'ownership-of-shares-more-than-25-percent-as-firm-registered-overseas-entity'
+}
+
+const natureTypeMap = new Map<string, string>([
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_SHARE_ENTITY, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_SHARE_ENTITY, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_ENTITY, BoTypes.BO_NATURE_OF_CONTROL],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_AS_TRUST, BoTypes.TRUST_NATURE_OF_CONTROL],
+  [controlMap.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL],
+  [controlMap.RIGHT_TO_APPOINT_AND_REMOVE_DIRECTORS_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL],
+  [controlMap.VOTING_RIGHT_MORE_THAN_25_PERCENT_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL],
+  [controlMap.OWNERSHIP_MORE_THAN_25_PERCENT_AS_FIRM, BoTypes.NON_LEGAL_NATURE_OF_CONTROL]
+]);
+
```

Comment:
Lines 140 to 154. Are these maps all the same now? Then can make them all the same map. NatureOfControlTypeMap?